planner/core/memtable_predicate_extractor.go

```diff
@@ -309,7 +331,8 @@ func (helper extractHelper) extractTimeRange(
 
 		var colName string
 		var datums []types.Datum
-		switch fn.FuncName.L {
+		fnName := helper.getTimeFunctionName(fn)
```

Comment:
The normal function will have the same signature with push-down function. How about:
```go
switch fn..Function.PbCode(){
}
```

Reply:
great, done.